src/backends/kimchi/mod.rs

```diff
@@ -243,12 +243,18 @@ impl KimchiVesta {
     }
 }
 
-#[derive(Default, Clone, Copy, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(Default, Clone, Copy, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, Ord)]
 pub struct KimchiCellVar {
     index: usize,
     pub span: Span,
 }
 
+impl PartialOrd for KimchiCellVar {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        Some(self.index.cmp(&other.index))
+    }
+}
+
```

Comment:
that's great! but can you use this approach instead? https://github.com/zksecurity/noname/pull/108

I think it would look like:

```diff
#[educe(PartialCmp)]
pub struct Ident {
    index: usize,
+    #[educe(PartialCmp(ignore))]
    pub span: Span,
}
```